src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml

```diff
@@ -13,7 +13,8 @@
 	</resultMap>
 
 	<sql id="ManagerToChannelColumnList">
-		id, property_manager_id, channel_partner_id, channel_partner_commission, funds_holder, net_rate, version
+		manager_to_channel.id, manager_to_channel.property_manager_id, manager_to_channel.channel_partner_id, manager_to_channel.channel_partner_commission,
```

Comment:
You can make 2 sql script one for join and one without, and use the one you need depending on the task
for example:
```
<sql id="columns">
	id, ...
</sql>
```
```
<sql id="columns_for_join">
	manager_to_channel.id (or with alias mct.id), ...
</sql>
```

Reply:
@vasylnikorych done. Separate sql script for join was created